### ISSUE TITLE
"View source on GitHub" can be broken while loading

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt.fyi#446]
https://staging.wpt.fyi/results/gamepad/idlharness.window.html?label=stable initially links to https://github.com/web-platform-tests/wpt/blob/master/gamepad/idlharness.window.html when loading, but after loading the link is updated to https://github.com/web-platform-tests/wpt/blob/master/gamepad/idlharness.window.js, which works.

The fixuping is slow enough that I encountered it without trying to stress test. Hiding the link until it's been updated would work.

**Comment by lukebjerring** [comment]
Hiding the link would actually cause a bit of a jank, which I'm not too happy causing. Maybe a non-link placeholder, like 

> Fetching GitHub source...

?

**Comment by foolip** [comment]
Yeah, that'd work. Just not adding an href attribute until we have the final URL would also do the trick, and probably less noticeable in the typical case where people aren't focuses on this link.
